### ISSUE TITLE
DrawerLayout different Behavior when drawerPosition is set to Right

### ISSUE THREAD

**Issue by SystemBlad** [kmagiera/react-native-gesture-handler#464]
### Current Behavior (also the BUG is presented in react-navigation)

In DrawerLayout component   when drawerPosition  property is set to Right the Drawer behavior is wrong due to: 
- The drawer is shown when user swipe from right to left in the middle of the screen instead of same gesture from the border.
- After drawer is opened and closed completely and user swipe from left border to right, the drawer shows a flash in the right side of the screen.

### Expected Behavior

The drawer behavior must be same as when the drawerPosition  property is set to Left:
- Show the drawer just when user swipes from the border.
- Drawer flash not will be shown when user swipes from the contrary border.

### How to reproduce

- Execute the demo project
- Go to "Gesture handler based DrawerLayout"
- Try to swipe in the middle of the screen from the right to left, you will see the first issue.
- After that close the drawer and try to swipe from the left border, you will see the drawer flash.

Demo Project:
https://drive.google.com/open?id=1DT086RShAASE4_lx3kzGbWfr9hTPl9uV

### Your Environment

| software         | version
| ---------------- | -------
| react-native-gesture-handler | 1.0.15
| react-native     | 0.58.4
| compileSdkVersion             | 28
| supportLibVersion      | 28.0.0

**Comment by adawi9** [comment]
I have the same issue...do you solve it?

**Comment by SystemBlad** [comment]
Hi @adawi9, I had fixed it by momentarily updating the file:
**node_modules \ react-navigation-drawer \ dist \ views \ DrawerView.js.**
you must replace the DrawerLayout component with DrawerLayoutAndroid.
I tested it only on Android and it's working.

**Comment by adawi9** [comment]
Can you share the updated file with me to replace it on my project

**Comment by SystemBlad** [comment]
Yes, the mentioned file is:
[DrawerView.zip](https://github.com/kmagiera/react-native-gesture-handler/files/2884656/DrawerView.zip)

The file must be located in the following path:
Your project\node_modules\react-navigation-drawer\dist\views\DrawerView.js

**Comment by adawi9** [comment]
Yeah it is working now...thanks dear...I think the team of react-native-gesture-handler and react-navigation must fix it...I attached a file to handle android and iOS
[DrawerView.js.zip](https://github.com/kmagiera/react-native-gesture-handler/files/2884701/DrawerView.js.zip)

**Comment by laukaichung** [comment]
Yes, I have the same problem. The edgeWidth prop is ignored when the sidebar is set to Right.

**Comment by Roshdy** [comment]
any update on this using react-navigation v3?

**Comment by computerjazz** [comment]
I just updated to `react-native-gesture-handler: "1.2.1"` and it solved the buggy drawer behavior.

**Comment by Aidurber** [comment]
Can confirm that >1.2.1 fixed it for me. I'm currently on 1.3.0 and it's working perfectly. This could probably be closed.
